test-cases/features/longevity-2mv-backpressure-4d.yaml

```diff
@@ -0,0 +1,15 @@
+test_duration: 6480
+
+prepare_write_cmd: "cassandra-stress user profile=/tmp/c-s_profile_2mv_2queries.yaml ops'(insert=1)' cl=QUORUM n=250000000 -port jmx=6868 -mode cql3 native -rate threads=200"
```

Comment:
`c-s_profile_2mv_2queries.yaml` is in our data_dir ? right ?

note to self in I think I've broke this in #1196

Reply:
@fruch  Yes, it's our user profile. I don't see this profile in your PR. Why do you think you broke it?

Reply:
I think I've broke the way it's assumes the location is in data_dir

Reply:
#1196 is not merged. Are you going to fix it?

Reply:
@juliayakovlev this comment not relevant to this PR, if Israel broke smth he will fix in his PR :)